docs/src/tutorial.md

```diff
@@ -0,0 +1,44 @@
+# Tutorial
+
+This tutorial is essentially a collection of examples.
+
+## Performance Profile
+
+Performance profiles are straightforward to use. The input is a matrix `T` with entries `T[i,j]` indicating the cost to solve problem `i` using solver `j`. Cost can be, for instance, elapsed time, or number of evaluations. The cost should be positive. Otherwise, and values will be transposed by 1.
```

Comment:
The last part should say "If any cost is zero, all measures will be shifted by 1."